lib/spandex_tesla.ex

```diff
@@ -48,48 +48,54 @@ defmodule SpandexTesla do
   @doc """
   Telemetry handler. Attach it to the telemetry tesla events in order to trace the tesla calls.
   """
-  def handle_event([:tesla, :request], measurements, metadata, _) do
+  def handle_event([:tesla, :request, :start], _measurements, _metadata, _config) do
     if tracer().current_trace_id([]) do
-      now = clock_adapter().system_time() |> System.convert_time_unit(:native, :nanosecond)
```

Comment:
Question, I remember to added this conversion for some reason. 

Can you confirm that it is really not necessary?

Reply:
Sure! That change was part of the Tesla 1.3.3 release as well. It was introduced in https://github.com/teamon/tesla/pull/324.

Before 1.3.3, Tesla used [`:timer.tc/3`](https://erlang.org/doc/man/timer.html#tc-3) which reports time in microseconds. The above PR made the change to [`System.monotonic_time/0`](https://hexdocs.pm/elixir/System.html#monotonic_time/0), which reports time in native units.